src/lis3dh-motion-detection.h

```diff
@@ -68,12 +70,13 @@ typedef enum
 {
 	DET_STOP,
 	DET_MOVE,
-	//...
+	BIKE_FALL,
 } event_t;
 
 typedef enum
 {
-	INT_1 = 1,
+	INT_0,
```

Comment:
INT_0 is not intuitive, I believe it's better to create another function to disable interupts

Reply:
Ok